### DISCUSSION TITLE
tsconfig settings

### DISCUSSION THREAD

**Discussion by dmeij** [typescript-language-server/typescript-language-server#928] [Q&A]
Hi,

Does the language server automatically use the tsconfig settings? I'm trying to use paths setting for import aliasses for @components and other folders. I'm using the monaco editor solution from monaco languageclient.

**Comment by rchl** [comment]
Yes, it does

**Comment by dmeij** [comment]
Is it correct that I don't see these path suggestions in the autocomplete?

**Reply by rchl** [reply]
I can't help you without some concrete example in form of a repo. There can be many reason for this not working. It should work.
